flaskr.py

```diff
@@ -80,4 +80,16 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
+	return redirect(url_for('show_entries'))
```

Comment:
General note: for some reason the code is over-indented, the base python style guide, [pep8](https://www.python.org/dev/peps/pep-0008/) requires 4 spaces.